### PR TITLE
Address couple of clang-tidy warnings

### PR DESCRIPTION
**Pull request by dimitry-**
Test: builds

### CODE REVIEW

File: src/OrbitGl/GraphTrack.h [hunk 1]
```diff
@@ -95,7 +96,6 @@ class GraphTrack : public Track {
   MultivariateTimeSeries<Dimension> series_;
 
  private:
-  [[nodiscard]] virtual std::string GetLegendTooltips(size_t /*legend_index*/) const { return ""; }
```
Comment (vickyliu-go4it) [inline src/OrbitGl/GraphTrack.h:98]: Why we are changing `GetLegendTooltips` from `private` to `protected`? We want this method to be used only in the `GraphTrack`, wouldn't it be better if it is `private`?

Also is there any advantage to make it purely virtual?

**Review by vickyliu-go4it — APPROVED** [review]
Thanks for addressing the clang-tidy warnings!

**Review by karupayun — APPROVED** [review]

File: src/OrbitGl/GraphTrack.h (hunk 1 above)
Comment (dimitry-) [reply src/OrbitGl/GraphTrack.h:98]: For some reason I was under impression private virtual method cannot be overridden. Will revert this to private - it is better.

as for pure virtual the advantage is compiler won't let you miss implementing this. It is my understanding that this method should be implemented by derived classes here as a rule not as an exception.

Comment (dimitry-) [reply src/OrbitGl/GraphTrack.h:98]: done

**Pull request merged**